README.rst

```diff
@@ -214,8 +214,8 @@ Choose from the list of available rules:
 
   Configuration options:
 
-  - ``syntax`` (``'long'``, ``'short'``): whether to use the ``long`` or ``short`` array
-    syntax; defaults to ``'long'``
+  - ``syntax`` (``'long'``, ``'short'``): whether to use the ``long`` or ``short`` array syntax;
```

Comment:
If changing the format is out of scope then the changes here should be minimized.
I.e. there is no need to change the line length here and so should be reverted.